frontend/src/service/utilities/Clipboard.service.ts

```diff
@@ -0,0 +1,44 @@
+export class ClipboardService {
+  private fallbackCopyTextToClipboard(text: string): boolean {
+    let textArea = document.createElement('textarea');
+
+    // set minimal visual effects in case textarea flashes
+    textArea.style.position = 'fixed';
+    textArea.style.top = '0';
+    textArea.style.left = '0';
+
+    textArea.style.width = '2em';
+    textArea.style.height = '2em';
+
+    textArea.style.padding = '0';
+    textArea.style.border = 'none';
+    textArea.style.outline = 'none';
+    textArea.style.boxShadow = 'none';
+    textArea.style.background = 'transparent';
+
+    textArea.value = text;
```

Comment:
How about doing this instead?
textArea.style.display = 'none';

Reply:
As per this method, the component should be there in the page to be able to select the text and copy to the clipboard. So, we are adding the component to the page, selecting, copying it and removing it from the page. Setting display to none won't let us select the text inside the textArea. The answer [here](https://stackoverflow.com/questions/400212/how-do-i-copy-to-the-clipboard-in-javascript) explains it in detail.

Reply:
I tested changing the style to `textArea.style.display = 'none';` as well. It didn't work.